pages/_app.js

```diff
@@ -0,0 +1,7 @@
+import '../styles/globals.css'
```

Comment:
Let's remove separate css files if possible.

Reply:
Importing globals.css is not automatic. It is required to have a imported css file for the tailwind directives.

See [Tailwind Next Guide](https://tailwindcss.com/docs/guides/nextjs) (step 4)